src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/dialogs/FlatWelcomeFrame.java

```diff
@@ -65,12 +66,12 @@ public void clearWorkspace() {
         // delete all the projects' links from the 'Welcome to IntelliJ IDEA' dialog
         int numberOfLinks = getNumberOfProjectLinks();
         for (int i = 0; i < numberOfLinks; i++) {
-            ComponentFixture recentProjectsList = find(ComponentFixture.class, byXpath("//div[@accessiblename='Recent Projects' and @class='MyList']"), Duration.ofSeconds(10));
+            JListFixture recentProjectsList = jLists(byXpath("//div[@accessiblename='Recent Projects']")).get(0);
```

Comment:
Should not there be NPE check on the next line?

Reply:
I do not think so, because there is the getNumberOfProjectLinks() method just before the for loop. If no LIST is available no iteration of loop will be made. Tested many many times with no List, with List with one item or with List with several items. Works without any issue.

Reply:
Fixed.

Reply:
hah, was too late